ha/core/config/config_manager.py

```diff
@@ -41,7 +41,7 @@ class ConfigManager:
     @staticmethod
     def init(log_name, log_path=None, level="INFO",
             backup_count=5, file_size_in_mb=10, syslog_server=None,
-            syslog_port=None, console_output=False, config_file=None):
+            syslog_port=None, console_output=True, config_file=None):
```

Comment:
This means that logs will always go to console now right  i.e. no configuration available to disable this.
Please confirm that this is  the expected behavior @shaileshsvc 
[In case of logging level debug console will get flooded.]

Reply:
Yes currently all log will go to console 

current py.util log and design dose not support to filter log for file and console
We can have separate design for this will create task to separate log

@satish-seagate @shaileshsvc

Reply:
CORTX-28795 created ticket